delete.php

```diff
@@ -0,0 +1,16 @@
+<?php
+require_once 'src/Models/CarsModel.php';
+
+if (isset($_GET['delete']))
+{
+    $db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
+    $cars_model = new CarsModel($db);
+    $id_to_delete = $_GET['delete'];
+    unset($_GET['delete']);
```

Comment:
Don't forget that unsetting $\_GET stuff doesn't actually do anything - dumb php